jira/client.py

```diff
@@ -320,6 +320,9 @@ def __init__(self, server=None, options=None, basic_auth=None, oauth=None, jwt=N
                 for name in f['clauseNames']:
                     self._fields[name] = f['id']
 
+        self.field_map = {}
```

Comment:
Minor: How about `self.field_map = self.generate_field_map()` for better readability?